### PR TITLE
Riskdev

### PR DESCRIPTION
**Pull request by LeonardoViotti**
Pull request to update master with currently published changes to the dashboard.

### CODE REVIEW

File: dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_subscribers_data_scaled.R [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+EXPORT <- T
+
+#### Load / Prep Percent Active
+percent_active_df <- read.csv(file.path(DATABRICKS_PATH, "indicator 4", "admin2", "percent_of_all_subscribers_active_option3_per_day.csv"),
+                              stringsAsFactors=F)
+percent_active_df <- percent_active_df %>% 
+  dplyr::rename(region = home_region) %>%
+  dplyr::select(region, day, percent_active)
+  
+unit <- "district"
```
Comment (ramarty) [inline dashboard-dataviz/zimbabwe_dashboard/01_preparing_data_for_dashboard/02_clean_telecom_data/clean_subscribers_data_scaled.R]:
```suggestion

```

**Review by ramarty — COMMENTED** [review]
helpful for testing code but not needed

**Comment by ramarty** [comment]
All looks good to me! @kbjarkefur This seems to just affect dashboard code and not anything elsewhere (except for slight addition to gitignore) -- but let me know if you want to review/check things here as well, then I can merge the pull request.

**Comment by kbjarkefur** [comment]
I took a quick look on file paths and searched for strings that could be related to sensitive info, and I found no reason not to merge this.

**Pull request merged**